examples/src/main/java/dev/braintrust/examples/ExperimentExample.java

```diff
@@ -37,10 +37,10 @@ public static void main(String[] args) throws Exception {
                         // will append new cases to
                         // the same experiment
                         .cases(
-                                EvalCase.of("strawberry", "fruit"),
-                                EvalCase.of("asparagus", "vegetable"),
-                                EvalCase.of("apple", "fruit"),
-                                EvalCase.of("banana", "fruit"))
```

Comment:
NOTE: `EvalCase` is still supported for backwards compatibility (I've marked the class as deprecated and encouraged use of `DatasetCase` instead).